content/blog/tidymodels-2025-q1/index.Rmd

```diff
@@ -0,0 +1,235 @@
+
+slug: tidymodels-2025-q1
+title: Q1 2025 tidymodels digest
+date: 2025-02-27
+author: Max Kuhn
+description: >
+    A summary of the goings on for the tidymodels group in later 2024 and early 2025.
+
+photo:
+  author: George Stagg and Max Kuhn
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [roundup] 
+tags: [tidymodels]
+---
+
+<!--
+TODO:
+* [ ] Look over / edit the post's title in the yaml
+* [ ] Edit (or delete) the description; note this appears in the Twitter card
+* [ ] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [ ] Find photo & update yaml metadata
+* [ ] Create `thumbnail-sq.jpg`; height and width should be equal
+* [ ] Create `thumbnail-wd.jpg`; width should be >5x height
+* [ ] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+The tidymodels framework is a collection of R packages for modeling and machine learning using tidyverse principles.
+
+Since the beginning of 2021, we have been publishing quarterly updates here on the tidyverse blog summarizing what’s new in the tidymodels ecosystem. The purpose of these regular posts is to share useful new features and any updates you may have missed. You can check out the tidymodels tag to find all tidymodels blog posts here, including our roundup posts as well as those that are more focused.
+
+We've recently sent a steady stream of tidymodels packages to CRAN recently. We usually release them in batches since many of our packages are tightly coupled with one another. Internally, this process is referred to as the "cascade" of CRAN submissions.  
+
+The post will update you on which packages have changed and the major improvements you should know. 
+
+Here's a list of the packages and their News sections: 
+
+- [baguette](https://baguette.tidymodels.org/news/index.html)
+- [brulee](https://brulee.tidymodels.org/news/index.html)
+- [censored](https://censored.tidymodels.org/news/index.html)
+- [dials](https://dials.tidymodels.org/news/index.html)
+- [hardhat](https://hardhat.tidymodels.org/news/index.html)
+- [parsnip](https://parsnip.tidymodels.org/news/index.html)
+- [recipes](https://recipes.tidymodels.org/news/index.html)
+- [tidymodels](https://tidymodels.tidymodels.org/news/index.html)
+- [tune](https://tune.tidymodels.org/news/index.html)
+- [workflows](https://workflows.tidymodels.org/news/index.html)
+
+Let's look at a few specific updates.
+
+## Improvements in Errors and Warnings
+
+A group effort was made to improve our error and warning messages across many packages. This started with an internal "upkeep week" (which ended up being 3-4 weeks) and concluded at the [Tidy Dev Day in Seattle](https://www.tidyverse.org/blog/2024/04/tdd-2024/) after Posit conf.  
+
+The goal was to use new tools in the cli and rlang packages to make messages more informative than they used to be. For example, using 
```

Comment:
There are a couple different styles in this post for concluding/beginning text before/after code chunks. I don't know that the group has standardized on any one. e.g. always ending in a colon for incomplete sentences or always referring to code chunks with a complete sentences would make this feel more polished

Reply:
Colons ok for you?